packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -146,18 +182,30 @@ class Iterable implements AbstractIterable<Opaque, IterationItem<Opaque>, Updata
       return array.length > 0 ? new ArrayIterator(array, keyFor) : EMPTY_ITERATOR;
     } else if (iterable === undefined || iterable === null) {
       return EMPTY_ITERATOR;
-    } else {
+    } else if (typeof iterable === 'object') {
+       let keys = Object.keys(iterable);
+       let kvPairs = keys.map(key => <KVPair>{ key, value: iterable[key] });
```

Comment:
Usually you wouldn't need to do an explicit cast like this. If it couldn't figure it out, something is probably wrong.